### PR TITLE
refactor(forms): provide RadioControlRegistry in root

### PR DESCRIPTION
**Pull request by sr5434**
I had to reopen #48738 becuase it closed automatically for some reason. @dylhunn approved the changes.

### CODE REVIEW

**Comment by JeanMeche** [comment]
Hi, your contribution still has some linting issues. See [the contributing guide](https://github.com/angular/angular/blob/main/CONTRIBUTING.md) to format your commit message as expected.

File: packages/forms/src/directives.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {NgModel} from './directives/ng_model';
 import {NgModelGroup} from './directives/ng_model_group';
 import {NgNoValidate} from './directives/ng_no_validate_directive';
 import {NumberValueAccessor} from './directives/number_value_accessor';
-import {RadioControlRegistryModule, RadioControlValueAccessor} from './directives/radio_control_value_accessor';
+import {RadioControlValueAccessor} from './directives/radio_control_value_accessor';
```
Comment (JeanMeche) [inline packages/forms/src/directives.ts:19]: You still need to remove the import on line 81.

Comment (sr5434) [reply packages/forms/src/directives.ts:19]: Is this better?

**Comment by JeanMeche** [comment]
You will need to update the golden files with `yarn run symbol-extractor:update`

Also your commit type should be lower case : `Refactor(forms)` => `refactor(forms)`

**Comment by sr5434** [comment]
I think I fixed it.

**Comment by JeanMeche** [comment]
> I think I fixed it.

Good job, a team member needs to approve it now and it will be merged 👍

**Comment by sr5434** [comment]
@dylhunn does my PR look good?

**Comment by sr5434** [comment]
Ok. Do you know when this PR will be approved?

**Comment by sr5434** [comment]
Why are the tests failing?

**Comment by dylhunn** [comment]
These are google internal tests. I'm running a fresh presubmit to find out (google link: http://cl/536744899)

**Review by dylhunn — APPROVED** [review]
reviewed-for: fw-forms, fw-core

pending g3 presubmit of course.

**Comment by sr5434** [comment]
Thanks

**Comment by JeanMeche** [comment]
Do you mind rebasing on the latest main ?

**Comment by sr5434** [comment]
Ok, sure

**Comment by sr5434** [comment]
I rebased

**Comment by JeanMeche** [comment]
There must have been a mistake during rebasing, the PR has 835 commits. Can you take a look at it ?

**Comment by JeanMeche** [comment]
I've picked up the improvement in #54130 and added you as co-author of the commit, we appreciate the initiative  👍

**Comment by sr5434** [comment]
Ok, great!

**Pull request closed without merging**